frontend/src/concepts/pipelines/context/PipelinesContext.tsx

```diff
@@ -308,10 +313,10 @@ export const PipelineServerTimedOut: React.FC = () => {
       }
     >
       <Stack hasGutter>
-        <StackItem>
-          We encountered an error creating or loading your pipeline server. To continue, delete this
-          pipeline server and create a new one. Deleting this pipeline server will delete all of its
-          resources, including pipelines, runs, and jobs.
+        <StackItem data-testid="timeout-pipeline-error-message">
+          {`We encountered an error creating or loading your pipeline server.${
+            errorMessage ? ` ${errorMessage}.` : ''
+          }`}
```

Comment:
we need to keep this text because it tells them how to recover and the consequences

>  We encountered an error creating or loading your pipeline server. To continue, delete this
          pipeline server and create a new one. Deleting this pipeline server will delete all of its
          resources, including pipelines, runs, and jobs.

So I would keep tat StackItem unchanged but add a stack item above that only renders if there is a error message

Reply:
Thanks made the changes and updated the screenshot accordingly